be/src/exec/vectorized/hdfs_scanner_text.cpp

```diff
@@ -191,7 +191,7 @@ Status HdfsTextScanner::parse_csv(int chunk_size, ChunkPtr* chunk) {
         int field_size = fields.size();
         if (_scanner_params.hive_column_names->size() != field_size) {
             LOG(WARNING) << strings::Substitute("Size mismatch between hive column $0 names and fields $1!",
```

Comment:
There will be a lot of logs when parsing old csv files.
Better to remove or use VLOG.